provider/datagen/src/baked_exporter.rs

```diff
@@ -733,3 +733,29 @@ impl BakedExporter {
         Ok(())
     }
 }
+
+macro_rules! cb {
+    ($($marker:path = $path:literal,)+ #[experimental] $($emarker:path = $epath:literal,)+) => {
+        fn key_to_marker_bake(key: DataKey, env: &databake::CrateEnv) -> databake::TokenStream {
+            use databake::Bake;
+            // This is a bit naughty, we need the marker's type, but we're actually
+            // baking its value. This works as long as all markers are unit structs.
```

Comment:
Aha, I was thinking whether we should make all our marker types be empty enums in v2, but here's a use case where we need them as a value

Reply:
I'm happy to redesign this, I don't like how we currently do it. The `#[data_struct]` macro could add a non-self `fn bake() -> TokenStream` to the marker.